docs/development/codeguide.rst

```diff
@@ -151,6 +153,19 @@ Coding Style/Conventions
     This should be done sparingly, and only
     when ``black`` produces undesirable formatting.
 
+      .. note::
+        When a list or array should be formatted as one item per line then this is best
+        achieved by using the
+        `magic trailing comma <https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma>`_.
+        This is frequently sufficient for formatting matrices while still allowing
+        ``black`` to check the code::
```

Comment:
```suggestion
        This is frequently sufficient for keeping matrices formatted as one row
        per line while still allowing ``black`` to check the code::
```
The only thing I've been stumbling on is that the trailing comma does not really trigger any new formatting here, it just tells black to preserve the existing one-per-line format – perhaps this is making it clearer.